import.java.uti.Scanner
class ExceptionDemo
{
    public static void main(String arr[])
    {
        int No1=0,No2=0,Ans=0;

        Scanner sobj = new Scanner(System.in);
        int Arr[5] = {10,20,30,40,50};
        int index = 0;
        System.out.println("Enter the first number :");
        No1 = sobj.nextInt();
        System.out.println("Enter the second number :");



    }
}